src/kernel/arch/aarch64/link.ld

```diff
@@ -1,8 +1,7 @@
 ENTRY(_start)
 
 SECTIONS {
-	/* The binary is loaded at 0x80000 for aarch64 and 0x8000 for arm */
-	. = 0x80000;
+	. = 0;
```

Comment:
Is this correct? I read that the Pi loads the kernel at 0x80000.

Reply:
config.txt kernel_old=1 will load it at  0
https://www.raspberrypi.org/documentation/configuration/config-txt/boot.md#kernel_old